<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="test_valid_fail_range_bytes.TestValidFailRangeBytes-20200818144243" tests="1" file="test_valid_fail_range_bytes.py" time="0.001" timestamp="2020-08-18T14:42:43" failures="0" errors="0" skipped="0">
	<testcase classname="test_valid_fail_range_bytes.TestValidFailRangeBytes" name="test_valid_fail_range_bytes" time="0.001" timestamp="2020-08-18T14:42:43" file="spec/python/test_valid_fail_range_bytes.py" line="9">
		<system-out><![CDATA[]]></system-out>
		<system-err><![CDATA[]]></system-err>
	</testcase>
</testsuite>
